src/App.tsx

```diff
@@ -239,6 +244,45 @@ const Profile = ({ user, userData, userName }) => {
   );
 };
 
+interface TrustLevelProperties {
+  trustLevel: number;
+  networkTime: number; //time on network in days
+}
+
+class TrustLevel extends React.Component<TrustLevelProperties, {}> {
```

Comment:
This can be a "functional component" (https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc), examples in App.tsx include `Profile` component. Probably wanna replace `computeTrustSuggestion` with its own `TrustSuggestion` component